pareto/utilities/get_data.py

```diff
@@ -17,32 +17,112 @@
 Authors: PARETO Team (Andres J. Calderon, Markus G. Drouven)
 """
 
+import logging
+import warnings
+from pathlib import Path
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import List
+from typing import Union
+
 import pandas as pd
 import requests
 import numpy as np
 import warnings
 
 
-def _read_data(_fname, _set_list, _parameter_list):
+_logger = logging.getLogger(__name__)
+
+
+try:
+    _dataframe_map = pd.DataFrame.map
+except AttributeError:
+    # compatibility with pandas 2.0.x
+    _dataframe_map = pd.DataFrame.applymap
+
+
+class DataLoadingError(ValueError):
+    def __init__(
+        self,
+        errors: Dict[str, Exception],
+    ):
+        self.errors = dict(errors)
+        self.summary = "\n".join(self._get_summary_lines())
+        super().__init__(self.summary)
+
+    def _get_summary_lines(self) -> List[str]:
+        max_name_len = max(len(name) for name in self.errors)
+
+        lines = ["Data loading failed for the following sheets: "]
+        for name, err in self.errors.items():
+            lines.append(f"\t{name:{max_name_len}}\t{err}")
+        return lines
```

Comment:
```suggestion
        lines.append("This may be because the sheets are empty (possibly excepting a header).")
        return lines
```
I think it's a good idea to explicitly list the most common cause of the error to ward off some potential user confusion.